docs/source/user_guide/app.rst

```diff
@@ -519,32 +519,26 @@ You can conveniently reset the sidebar groups to their default state by setting
     property, these fields will be dynamically assigned to default groups in
     the App at runtime.
 
-.. _app-lightning-mode:
+.. _app-query-performance-mode:
 
-Lightning mode
---------------
+Query Performance mode
+----------------------
 
-Lightning mode is a performant sidebar setting for larger datasets that can be
-enabled either by adding a `lightning_threshold` to your
+Query Performance mode is a performant sidebar setting for larger datasets that can be
+enabled either by adding a `default_query_performance` to your
 :ref:`App config <configuring-fiftyone-app>`, or for a particular dataset by
 clicking on the "Gear" icon above the sample grid in the App.
 
 .. note::
 
-    When lightning mode is enabled through the "Gear" icon in the App, the
+    When query performance mode is enabled through the "Gear" icon in the App, the
```

Comment:
Do we want to keep Query Performance or query performance for consistency?

Reply:
Good point. Note the language can and may change after product feedback